src/e2e-test/features/salesforcebatchsource/RunTime.feature

```diff
@@ -53,10 +53,9 @@ Feature: Salesforce Batch Source - Run time Scenarios
     And Open and capture logs
     And Verify the pipeline status is "Succeeded"
     And Close the pipeline logs
-    Then Validate the values of records transferred from Salesforce to Bigquery is equal
```

Comment:
why are these validations removed in existing tests?

Reply:
added done @14d27c02149dffbec08083e7dced9c752115b8c1